core/utils.py

```diff
@@ -26,7 +26,7 @@
 
 logger = logging.getLogger(__file__)
 
-CACHE_TIMEOUT = 3600 * 24
+
```

Comment:
Given the default value in the depending PR is now 3600, which is different from the value before. I don't have any problem with it. Just wanted to flag that it is different in case it's not intentional.